### PR TITLE
User home

### PR DESCRIPTION
**Pull request by max-friedman**
- Created way to open up documents that already exist
- Changed file type of user-home from html --> jsp

### CODE REVIEW

File: project/src/main/java/com/google/sps/Servlets/DocumentServlet.java [hunk 1]
```diff
@@ -21,6 +21,6 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String hash = request.getParameter("documentHash");
     request.setAttribute("documentHash", hash);
 
-    request.getRequestDispatcher("/document.jsp#-" + hash).forward(request, response);
+    request.getRequestDispatcher("/document.jsp#" + hash).forward(request, response);
```
Comment (tin0d0) [inline project/src/main/java/com/google/sps/Servlets/DocumentServlet.java:28]: was this the problem with not being able to load the correct doc?

**Review by tin0d0 — APPROVED** [review]
LGTM

File: project/src/main/java/com/google/sps/Servlets/DocumentServlet.java (hunk 1 above)
Comment (max-friedman) [reply project/src/main/java/com/google/sps/Servlets/DocumentServlet.java:28]: Yeah the hash includes the '-' so this was causing it to not find the hash

File: project/src/main/webapp/user-home.jsp [hunk 2]
```diff
@@ -19,5 +29,11 @@
         <docs-component></docs-component>
       </div>
     </div>
+
+    <script>
+      function loadDocument() {
+        window.location.href = "/Document?documentHash=" /* documentHash */;
```
Comment (g-luo) [inline project/src/main/webapp/user-home.jsp]: modify this to use the documentHash url attribute; and probably add a null check where if the hash doesn't exist, redirect back to user home

**Review by g-luo — APPROVED** [review]
lgtm

**Review by frankpzn — APPROVED** [review]

**Review by HidyHan — APPROVED** [review]

**Pull request merged**